docs/pdfs.md

```diff
@@ -6,7 +6,9 @@ slug: /pdfs
 ---
 
 ## Add labels and fields
-We recommend using [the Documate online free PDF editor](https://www.documate.org/pdf). It can do everything that AssemblyLine makes use of as far as PDF labels and fields go. Documate has [a page that talks about its PDF editor](https://documate.wrkflows.io/).
+We recommend you use [specific names for labels](docs_vars_reference.md) to take full advantage of the tools we've created.
```

Comment:
```suggestion
We recommend you use [specific names for labels](doc_vars_reference.md) to take full advantage of the tools we've created.
```

Reply:
Awesome! How the heck did you find that?!

Reply:
I didn't actually, it was docusaurus! I ran `npm run start`, and showed me this error:
```

✔ Client
  Compiled successfully in 582.98ms

warn Docs markdown link couldn't be resolved: (docs_vars_reference.md) in /home/brycew/Developer/LITLab/docassemble-AssemblyLine-documentation/docs/pdfs.md for version current


```